### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by rchrd2** [jacksingleton/accounts-sandstorm-dev#2]
Hi Jack,

This library sounds very useful. Unfortunately it's not working for me, and sadly, I don't know enough about Meteor to know why it is not working.

I can say this: Accounts do work in the `vagrant-spk dev` environment. But this library isn't working with the command `meteor`.

Do you know of anyway I can enable more verbose messages to debug this?

**Comment by jacksingleton** [comment]
Are you doing something like this to publish the sandstorm permissions to the client code?

https://github.com/jacksingleton/accounts-sandstorm-dev/blob/master/pubsub-sandstorm-permissions.js

**Comment by rchrd2** [comment]
Yeah, I actually just set that up so the client can access the permissions. It seems that Meteor on the server and client has no logged in user at all. Like it just isn't working.

The user code works and grabs the user account when running via `vagrant-spk dev`.

**Comment by rchrd2** [comment]
I'll try to dig in more and see if I can find the problem. Let me know if you have any ideas.

**Comment by jacksingleton** [comment]
Hmm the new version might be broken give 0.0.2 a quick try

**Comment by rchrd2** [comment]
Ah yes! This older version works! I got this in my console:

```
I20151005-19:12:33.062(-7)? { type: 'sandstorm', userId: '2D32aC7g2oTtCrRwj' }
```

**Comment by jacksingleton** [comment]
Okay my bad just pushed a fixed version, try 0.1.1

**Comment by rchrd2** [comment]
Great. This update works. Thanks for the fix.

 I've noticed another issue with the ENV variables, but I'll post it in a new issue.
